ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -671,7 +718,8 @@ def _generate_physical_disks(self):
                        autospec=True)
     def test_create_configuration(
             self, mock_commit_config, mock_validate_job_queue,
-            mock_list_physical_disks, mock_get_drac_client):
```

Comment:
Can you undo this change since it's not directly related to this patch?

Reply:
Replaced as is.